terraform/projects/app-ecs-services/config/alerts/data-gov-uk.yml

```diff
@@ -32,3 +32,11 @@ groups:
     annotations:
         summary: "Index size of Elasticsearch for {{ $labels.job }} has decreased significantly"
         description: "The index size of Elasticsearch for {{ $labels.job }} has decreased by more than 100 documents in the last 30 minutes"
+  - alert: DataGovUk_HighSidekiqEnqueuedJobs
+    expr: sidekiq_enqueued_jobs{org="gds-data-discovery",job="publish-data-production-queue-monitor"} > 800
+    for: 5m
+    labels:
+        product: "data-gov-uk"
+    annotations:
+        summary: "Sidekiq's enqueued jobs do not seem to be clearing for {{ $labels.app }}"
```

Comment:
`{{ $labels.app }}` will not evaluate to anything as you do not have a label on your metric with the key `app`.

You have the following labels available:
`sidekiq_enqueued_jobs{cf_app_instance="0",instance="7c027d9f-5339-4d93-bb42-d39a9470f657:0",job="publish-data-production-queue-monitor",org="gds-data-discovery",space="data-gov-uk"}`

You could instead use the `job` label or you could just hardcode production (or whatever you'd like to put in it's place).

It's all looking good apart from that!